store/blocks/header/top-menu.json

```diff
@@ -20,21 +20,6 @@
       }
     }
   },
-  "menu-item#about-us": {
```

Comment:
Why did you remove this?

Reply:
This block is duplicated

Reply:
This is duplicated: https://github.com/vtex-apps/store-theme/blob/master/store/blocks/header/category-menu.json#L155